sdcm/cluster.py

```diff
@@ -2752,6 +2749,26 @@ def get_scylla_config_param(self, config_param_name, verbose=True):
             self.log.error(f'Failed to retreive value of {config_param_name} parameter. Error: {e}')
             return None
 
+    def install_epel(self):
+        """
+        Standard repositories might not provide all the packages that can be installed on CentOS, RHEL,
+        or Amazon Linux-based distributions. Enabling the EPEL repository provides additional options for
+        package installation.
+        """
+        if not self.distro.is_rhel_like:
+            raise Exception('EPEL can only be installed for RHEL like distros')
+
+        if self.distro.is_amazon2:
+            # Enable amazon2-extras repo for installing epel
+            # Reference: https://aws.amazon.com/amazon-linux-2/faqs/#Amazon_Linux_Extras
+            self.remoter.run('sudo amazon-linux-extras install epel')
+
+        if self.distro.is_rhel8:
```

Comment:
Just to be sure, in case it's Amazon Linux-based distribution you would still run:
self.remoter.run('sudo amazon-linux-extras install epel')
self.remoter.run('sudo yum install -y epel-release', retry=3)

Reply:
Yes,   'sudo amazon-linux-extras install epel' only added a repo file, but epel-release package isn't installed.